ProjectMacros.cmake

```diff
@@ -394,17 +394,34 @@ macro(MAKE_SWIG_TARGET NAME SIMPLENAME KEY_I_FILE I_FILES PARENT_TARGET PARENT_S
     )
     set_source_files_properties(${COPY_PYTHON_GENERATED_SRC} PROPERTIES GENERATED TRUE)
 
-    install(FILES "${PYTHON_GENERATED_SRC}" DESTINATION Python COMPONENT "Python")
-    install(TARGETS ${swig_target} DESTINATION Python COMPONENT "Python")
+    # In build dir it's inside Products/python and we want to find Products/libopenstudiolib
+    # It installs in Python/ and we want to find lib/libopenstudiolib
+    if(APPLE)
+      set_target_properties(${swig_target}
+        PROPERTIES
+          SUFFIX ".so"
+          BUILD_RPATH "@loader_path;@loader_path/../"
+          INSTALL_RPATH "@loader_path;@loader_path/../lib/"
+      )
+    elseif(UNIX)
+      set_target_properties(${swig_target}
+        PROPERTIES
+          BUILD_RPATH "$ORIGIN:$ORIGIN/../"
+          INSTALL_RPATH "$ORIGIN:$ORIGIN/../lib/"
+      )
+    endif()
```

Comment:
Python bindings. Tweak rpaths to point to the ../lib/ folder. I'm not sure if it's really needed. On windows it works as is...